index.js

```diff
@@ -118,7 +118,14 @@ module.exports = {
       ['/color/', 'fill', 'stroke', 'font-size', 'line-height'],
       {
         ignoreKeywords: {
-          '/color/': ['currentColor', 'transparent', 'inherit'],
+          '/color/': [
+            'currentColor',
+            'transparent',
+            'inherit',
+            'underline',
+            'overline',
+            'line-through',
```

Comment:
Without this linter shows this error: `Expected variable, function or keyword for "underline" of "text-decoration-color" (scale-unlimited/declaration-strict-value)` due to [`text-decoration: underline`](https://github.com/Skyscanner/stylelint-config-skyscanner/blob/2f5a28094e8b45b40f0a1e0adc62308ed681b071/index.test.js#L71).
I guess it's because `text-decoration` is a shorthand for `text-decoration-line text-decoration-color text-decoration-style` and the plugin `stylelint-declaration-strict-value` might think the value of `text-decoration-line` must be applied to `text-decoration-color` also.

Do you think there's a better way to prevent this error?

Reply:
Support for shorthands has been added: https://github.com/AndyOGo/stylelint-declaration-strict-value#css-shorthand-syntax
This looks like a bug there :-( (I've created an [issue for them](https://github.com/AndyOGo/stylelint-declaration-strict-value/issues/68)).

We could disable expanding shorthands, but I think it's worth keeping it.
We could also try to disable this rule for `text-decoration-color` but I also think it's better to keep it enabled.

What do you think?

Reply:
`stylelint-declaration-strict-value` has been updated to disable shorthands support by default (to prevent this issue).
Would you enable it or keep it disabled? I like having it enabled, as it can detect more issues, but the solution to prevent the issue with `text-decoration` seems a bit hacky... 🤔

Reply:
Agreed with @olliecurtis to keep it disabled for now